src/games/strategy/triplea/ai/proAI/util/ProAttackOptionsUtils.java

```diff
@@ -396,6 +396,18 @@ public void findAttackOptions(final PlayerID player, final List<Territory> myUni
       final Map<Territory, Set<Territory>> landRoutesMap, final List<ProAmphibData> transportMapList,
       final List<Territory> enemyTerritories, final List<Territory> territoriesToCheck,
       final boolean isCheckingEnemyAttacks, final boolean isIgnoringRelationships) {
+    findAttackOptions(player, myUnitTerritories, moveMap, unitMoveMap, transportMoveMap, bombardMap, landRoutesMap,
+        transportMapList, enemyTerritories, new ArrayList<Territory>(), territoriesToCheck, isCheckingEnemyAttacks,
+        isIgnoringRelationships);
+  }
+
+  public void findAttackOptions(final PlayerID player, final List<Territory> myUnitTerritories,
```

Comment:
Agree. I ended up already doing this in the next set of changes. Really just did this to avoid having to update any references that called the original method without the extra parameter. I'd prefer to just merge this one as I have lots of refactoring changes already made in my next PR.